subsys/llext/llext_mem.c

```diff
@@ -191,7 +191,7 @@ int llext_copy_regions(struct llext_loader *ldr, struct llext *ext,
 
 void llext_adjust_mmu_permissions(struct llext *ext)
 {
-#ifdef CONFIG_MMU
+#if defined(CONFIG_MMU) && !defined(CONFIG_PANTHERLAKE)
```

Comment:
that is not a Zephyr symbol and we should not be checking for HW specific configs in subsys code...

Reply:
@nashif agree, this is a PoC, to test https://github.com/thesofproject/sof/pull/9886 , we need a proper solution for this

Reply:
for a record: here's what it looks like without this patch: https://sof-ci.01.org/sofpr/PR9886/build11387/devicetest/index.html